fixtures/README.md

```diff
@@ -0,0 +1,36 @@
+# Fixtures
+
+This folder contains sample projects we use in the integration and acceptance tests. 
+
+## app_with_tests
+
+Simple app with tests.
+
+## sample_2
+
+Slightly more complicated project consists of an iOS app and few frameworks. 
+
+#### Structure
+
+```
+Workspace:
+  - App:
+    - MainApp (iOS app)
+    - MainAppTests (iOS unit tests)
+  - Framework1:
+    - Framework1 (dynamic iOS framework)
+    - Framework1Tests (iOS unit tests)
+  - Framework2:
+    - Framework2 (dynamic iOS framework)
+    - Framework2Tests (iOS unit tests)
+```
+
+Dependencies:
+  - App -> Framework1
+  - App -> Framework2
+  - Framework1 -> Framework2
+
+
+## sample_3
```

Comment:
I noticed that the fixtures are named `sample_2`, `sample_3`.. etc.
Is there a reason for this over using more descriptive names or creating a folder hierarchy?

Reply:
I read the issue: https://github.com/tuist/tuist/issues/197#issuecomment-455500886 which describes some of your reasonings. I'm not sure on having the simplified incremental names is better than having the more verbose ones. 

However, having said that ... at the moment I don't like either.

Reply:
> we don't need to be very strict, it's OK to have some gaps like sample_3, sample_4, sample_7 so people don't need to necessarily pick up +1 number.

Sure, this makes sense. We need to make it clear that they are only used for identification purposes.

> we can update the main README first and land it without any fixture files, i.e. with [WIP] sample_4 and short description so others know that this name is already taken.

This will be a little harder to police - but worth a shot.

Have you seen the buck build test data repo? https://github.com/facebook/buck/tree/a6c7d7391a7a8e575c6372cac6690a240b5907ea/test/com/facebook/buck/apple/testdata

Reply:
@ollieatkinson wow, buck maintainers did really good job - I like it! I'm overthinking this, nice prefixes/consistent naming convention and reusability of the fixture projects should be enough to keep the structure tidy.
Let's stick with meaningful folder names - I will push the fix.